src/Couleur.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace IPSSI\POO;
+
+class Couleur
+{
+    public function __construct(string $couleur) {
```

Comment:
```suggestion
    public function __construct(int $r, int $g, int $b) {
```
et vérifier que ces valeurs sont entre 0 et 255